packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -105,7 +106,9 @@ class PictureLayer extends Layer {
 
   @override
   void addToScene(ui.SceneBuilder builder, Offset layerOffset) {
-    builder.addPicture(layerOffset, picture);
+    Timeline.timeSync('PictureLayer::addToScene', () {
+      builder.addPicture(layerOffset, picture);
+    });
```

Comment:
Generally we prefer the non-callback versions of these APIs, i.e., `Timeline.startSync` and `Timeline.finsihSync`.